README.md

```diff
@@ -21,11 +21,28 @@ You can install Ferrite from the Pkg REPL:
 pkg> add Ferrite
 ```
 
-## Contributing and Questions
-If you have specific questions about Ferrite.jl, you're welcome to reach out to us on the Julia Slack under #ferrite-fem or on Zulip under #Ferrite.jl ([here](https://julialang.org/community/) you can find links to Slack and Zulip). If you encounter a problem, please open an [issue](https://github.com/Ferrite-FEM/Ferrite.jl/issues). Feel free to ask us in case you are not sure if a problem is on the side of Ferrite.jl!  
-Contributions are very welcome, as are feature requests and suggestions.  
-Please keep in mind that we are part of the Julia community and adhere to the [Julia Community Standards](https://julialang.org/community/standards/).
+## Contributing
+
+See [CONTRIBUTING](CONTRIBUTING.md).
+
+## Questions
+
+If you have questions about Ferrite.jl you're welcome to reach out to us on the Julia Slack
+under `#ferrite-fem` or on Zulip under `#Ferrite.jl` ([here][community] you can find links
+to Slack and Zulip). Alternatively you can start a [new discussion][gh-discussion] in the
+discussion forum on the repository. Feel free to ask us even if you are not sure the problem
+is with Ferrite.jl.
+
+If you encounter what you think is a bug please report it, see
+[CONTRIBUTING.md](CONTRIBUTING.md#reporting-issues) for more information.
+
```

Comment:
```suggestion

## Community Standards
```
The below applies to both Contributing and Questions, so I wouldn't put it under only one of them.